config/initializers/image_adapter.rb

```diff
@@ -0,0 +1 @@
+ImageAdapter::Resolver.endpoint = Djatoka::Resolver.new(Settings.stacks.djatoka.url)
```

Comment:
This creates the initial resolver. I have found that an error eventually will cause this to be nil, so further testing needs to be included in case it may need to be reset at runtime.